README.md

```diff
@@ -53,7 +53,7 @@ any new files in the repository.
 Make your own customizations
 ----------------------------
 
-Put your customizations in dotfiles appended with `.local`:
+Put your customizations in `~/dotfiles-local` appended with `.local`:
```

Comment:
Since there are a few methods to maintaining customizations, what about having a section "Maintaining your customizations" that explains forking the dotfiles, a dotfiles-local and just managing the changes locally without a vcs?